action.yml

```
@@ -11,6 +11,10 @@ inputs:
     description: 'Packages required at build time, space-separated'
```

For clarity, we should specify this option as "pip packages". This also needs to be changed in `README.md`
```suggestion
    description: 'pip packages required at build time, space-separated'
```